proto/penumbra/penumbra/core/component/auction/v1alpha1/auction.proto

```diff
@@ -14,8 +14,24 @@ message GenesisContent {
   AuctionParameters params = 1;
 }
 
-// Query operations for the Auction component.
-service QueryService {}
+// Query operations for the auction component.
+service QueryService {
+  // Get the current state of an auction by ID.
+  rpc AuctionStateById(AuctionStateByIdRequest) returns (AuctionStateByIdResponse);
+}
+
+message AuctionStateByIdRequest {
+    AuctionId id = 1;
+}
+
+message AuctionStateByIdResponse {
+    // If present, the state of the auction. If not present, no such auction is known.
+    DutchAuctionState auction = 2;
```

Comment:
We should change this to an `Any` type and let clients inspect/deserialize to the concrete auction that the type mentions. The goal of this RPC is to be a minimal/general purpose "get me the auction state".

Reply:
changed to `google.protobuf.Any`